start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
Maybe I am misunderstanding something. To me, it seems, this will fail with an error when not run by travis. Boolean isTravis is undeclared in that case. Maybe put this in a try/except clause. But this also seems to not be the proper way to do it.